istioctl/pkg/multicluster/remote_secret.go

```diff
@@ -550,6 +535,7 @@ func createRemoteSecret(opt RemoteSecretOptions, client kube.ExtendedClient, env
 		return nil, err
 	}
 
+	remoteSecret.Annotations["topology.istio.io/clusterType"] = opt.Type.String()
```

Comment:
not sure if this annotation should go into api

Reply:
If customers have to set this, I think it might need to be documented some where - probably api is the best place. Can you please extract this to a constant?

Reply:
But looking at the code, it looks like we set it while creating  it so may be not needed in api?